<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsciiStringDialogClass</class>
 <widget class="QDialog" name="AsciiStringDialogClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit non-unicode string</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../analogexif.qrc">
    <normaloff>:/images/icons/film_edit.png</normaloff>:/images/icons/film_edit.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>You have entered non-Ascii characters into this field. Some metadata tags associated with this field do not support Unicode and can not store these characters properly. The value you have entered will be converted to Ascii according to your computer's locale; however, you can review the result and provide Ascii-only alternative to the entered value.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Unicode value</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="unicodeEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Ascii value</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="asciiEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QToolButton" name="copyBtn">
       <property name="toolTip">
        <string>Copy and convert text from Unicode to Ascii</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../analogexif.qrc">
         <normaloff>:/images/icons/arrow_undo.png</normaloff>:/images/icons/arrow_undo.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../analogexif.qrc"/>
 </resources>
 <connections/>
</ui>
